digitaltwin_client/samples/readme.md

```diff
@@ -4,24 +4,79 @@ This directory contains samples demonstrating creating and using Digital Twin in
 
 ## Directory contents
 
-* [digitaltwin_sample_device](./digitaltwin_sample_device) - executable that interacts with the other Digital Twin interfaces in this directory.
-* [digitaltwin_sample_ll_device](./digitaltwin_sample_ll_device) - executable that interacts with the other Digital Twin interfaces in this directory, using the \_LL\_ layer and enabling Device Provisioning Service (DPS).
-* [digitaltwin_sample_device_info](./digitaltwin_sample_device_info) - library that implements a sample device info Digital Twin interface.  This interface reports information about the device - such as OS version, amount of storage, , etc.
-* [digitaltwin_sample_environmental_sensor](./digitaltwin_sample_environmental_sensor) library that implements a sample environmental sensor Digital Twin interface.  This interface demonstrates all concepts of the implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
+### Executable Source Code
+* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using Iot Hub.
+* [digitaltwin\_sample\_ll_device](./digitaltwin_sample_ll_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation interfaces and basic operation.  It uses the lower level (\_LL\_) layer and enables the Device Provisioning Service (DPS) to connect to Iot Central. It is appropriate for devices with limited resources.
+
+### Libraries
+* [digitaltwin\_sample\_device_info](./digitaltwin_sample_device_info) - Library that implements a sample device info Digital Twin interface of helper functions.  This interface reports information about the device - such as OS version, amount of storage, etc.
+* [digitaltwin\_sample\_environmental_sensor](./digitaltwin_sample_environmental_sensor) - Library that implements a sample environmental sensor Digital Twin interface of helper functions.  This interface demonstrates all concepts of implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
 
 ## Getting started
+Build the Digital Twin Client SDK, if not already completed.  Follow instructions [here](../doc/building_sdk.md).
+
+### digitaltwin\_sample\_device
+
+* Use cmake to build the [digitaltwin\_sample\_device](digitaltwin_sample_device) directory.
+
+  From the azure-iot-c-sdk repository root:
+  ```
+  cd cmake
```

Comment:
This assumes they have already done the
```bash
cmake -D<whatever-the-option-is-for-digital-twin> ..
```
correct?

Reply:
Correct.  That step is covered in the "Build the Digital Twin Client SDK" instruction link on line 16.  (/digitaltwin_client/doc/building_sdk.md)